tensorpack/utils/utils.py

```diff
@@ -91,3 +93,62 @@ def get_tqdm(**kwargs):
 def building_rtfd():
     return os.environ.get('READTHEDOCS') == 'True' \
         or os.environ.get('TENSORPACK_DOC_BUILDING')
+
+
+def intensity_to_rgb(intensity, cmap='Blues_r'):
+    """Convert a matrix of intensities to an rgb image employing a colormap.
+
+    Nice colormaps are:
+      - Blues, Blues_r
+      - Reds, Reds_r
+      - BuGns, BuGns_r
+      - Greys, Greys_r
+
+    Args:
+        intensity (TYPE): array of intensities such as saliency
+        background (None, optional): background for heatmap
+        cmap (str, optional): used colormap (required matplotlib)
+
+    Returns:
+        TYPE: nice heatmap
```

Comment:
Could you document the expected format of input/output image? Usually I will be especially confused with the range (0-1 or 0-255?). RGB/BGR is not a big deal in this case, though.